### PR TITLE
Deploying sTLSA

### PR DESCRIPTION
**Pull request by SynthAman**

### CODE REVIEW

File: fas [hunk 1]
```diff
@@ -0,0 +1,2252 @@
+[33mcommit 73d94c51a14f763f601607dffe843f64c2b0faab[m[33m ([m[1;36mHEAD -> [m[1;32mmaster[m[33m, [m[1;33mtag: v2.38.0[m[33m, [m[1;31morigin/staging[m[33m, [m[1;31morigin/master[m[33m)[m
```
Comment (justinjmoses) [inline fas:1]: Please remove this file

File: package-lock.json [hunk 2]
```diff
@@ -547,9 +547,9 @@
 			},
 			"dependencies": {
 				"@ethersproject/abi": {
-					"version": "5.0.11",
-					"resolved": "https://registry.npmjs.org/@ethersproject/abi/-/abi-5.0.11.tgz",
-					"integrity": "sha512-ibZswQsjdFuLSfY2lbRTZM2Uk+ci7tp+mjVK0kjxVol2V32cb7va1r6B4AJU/Ac/VTstCjxtn0KKMfbkPc002w==",
+					"version": "5.0.12",
```
Comment (justinjmoses) [inline package-lock.json:550]: please remove this file change from the PR

File: publish/deployed/mainnet/owner-actions.json [hunk 3]
```diff
@@ -1 +1,9 @@
-{}
+{
```
Comment (justinjmoses) [inline publish/deployed/mainnet/owner-actions.json:1]: please undo this change in the PR

**Review by justinjmoses — CHANGES_REQUESTED** [review]
Thanks SynthAman. Can you please remove the extra file and undo the changes to owner actions and package-lock and push that? Also the base has been changed to the `develop` branch

**Comment by SynthAman** [comment]
Yeah sure, I will do it now

On 2/10/2021 9:58:25 AM, justin j. moses <notifications@github.com> wrote:
@justinjmoses requested changes on this pull request.
Thanks SynthAman. Can you please remove the extra file and undo the changes to owner actions and package-lock and push that? Also the base has been changed to the develop branch
In fas [https://github.com/Synthetixio/synthetix/pull/1077#discussion_r573793161]:
> @@ -0,0 +1,2252 @@ +[33mcommit 73d94c51a14f763f601607dffe843f64c2b0faab[m[33m ([m[1;36mHEAD -> [m[1;32mmaster[m[33m, [m[1;33mtag: v2.38.0[m[33m, [m[1;31morigin/staging[m[33m, [m[1;31morigin/master[m[33m)[m
Please remove this file
In package-lock.json [https://github.com/Synthetixio/synthetix/pull/1077#discussion_r573793449]:
> @@ -547,9 +547,9 @@ }, "dependencies": { "@ethersproject/abi": { - "version": "5.0.11", - "resolved": "https://registry.npmjs.org/@ethersproject/abi/-/abi-5.0.11.tgz", - "integrity": "sha512-ibZswQsjdFuLSfY2lbRTZM2Uk+ci7tp+mjVK0kjxVol2V32cb7va1r6B4AJU/Ac/VTstCjxtn0KKMfbkPc002w==", + "version": "5.0.12",
please remove this file change from the PR
In publish/deployed/mainnet/owner-actions.json [https://github.com/Synthetixio/synthetix/pull/1077#discussion_r573794493]:
> @@ -1 +1,9 @@ -{} +{
please undo this change in the PR

**Review by justinjmoses — APPROVED** [review]
Approved into intermediary branch.
